### PR TITLE
Site Settings: Update texts in toggle descriptions and InfoPopover content in Media Settings card

### PR DESCRIPTION
**Pull request by oskosk**
* This PR updates some texts in the Media Setting Card for Jetpack Sites.
* It also updates the description text for the Photon toggle from `Enabling Photon is required to use Tiled Galleries` to `Must be enabled to use tiled galleries.`

PR #11906 removed some periods from toggle labels and descriptions. The latter was not desired. 

#### Testing instructions

1. Check the [Writing Settings page in the live branch](https://calypso.live/settings/writing?branch=fix/media-settings-card-labels) for one of your Jetpack sites
1. Get to the Media Settings Card
1. Confirm that the usage of periods for the texts is the expected.

#### After this PR

image

#### Before this PR

image

#### Why

During #11906's review, It was recommended that the labels action texts don't use ending periods. And that the descriptions and the InfoPopovers content, use punctuation appropriately.

### CODE REVIEW

File: client/my-sites/site-settings/media-settings/index.jsx [hunk 1]
```diff
@@ -47,23 +47,23 @@ const MediaSettings = ( {
...
+					description={ translate( 'Enabling Photon is required to use Tiled Galleries.' ) }
 					disabled={ isRequestingSettings || isSavingSettings || photonModuleUnavailable }
 					/>
 			</FormFieldset>
 			<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
 				<div className="media-settings__info-link-container site-settings__info-link-container">
 					<InfoPopover position={ 'left' }>
 						<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/carousel' } >
-							{ translate( 'Learn more about Carousel' ) }
+							{ translate( 'Learn more about Carousel.' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/media-settings/index.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/site-settings/media-settings/index.jsx [hunk 2]
```diff
@@ -47,23 +47,23 @@ const MediaSettings = ( {
...
+							{ translate( 'Learn more about Photon.' ) }
 						</ExternalLink>
 					</InfoPopover>
 				</div>
 				<JetpackModuleToggle
 					siteId={ siteId }
 					moduleSlug="photon"
 					label={ translate( 'Speed up your images and photos with Photon' ) }
-					description={ translate( 'Enabling Photon is required to use Tiled Galleries' ) }
+					description={ translate( 'Enabling Photon is required to use Tiled Galleries.' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/media-settings/index.jsx]: :ok: This change is safe, we'll keep the existing 1 translations.

File: client/my-sites/site-settings/media-settings/index.jsx [hunk 3]
```diff
@@ -47,23 +47,23 @@ const MediaSettings = ( {
 				<div className="media-settings__info-link-container site-settings__info-link-container">
 					<InfoPopover position={ 'left' }>
 						<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/photon' } >
-							{ translate( 'Learn more about Photon' ) }
+							{ translate( 'Learn more about Photon.' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/media-settings/index.jsx]: :ok: This change is safe, we'll keep the existing 18 translations.

**Comment by oskosk** [comment]
cc @MichaelArestad

**Comment by MichaelArestad** [comment]
LGTM

**Pull request merged**